template/sample/check/CheckTesterProcessor.gen.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 Istio Authors.
```

Comment:
There is nothing generated for us to call at config validation time, like with traditional adapters.

https://github.com/istio/mixer/blob/master/pkg/adapter/adapter.go#L49